### DISCUSSION TITLE
Extra Reading Response

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#132] [Extra Credit]
# Extra Reading Response
### 20 points

Read the following paper:

[Hirst, M. (2012) One tweet does not a revolution make: Technological determinism, media and social change. Global Media Journal.](https://www.hca.westernsydney.edu.au/gmjau/archive/v6_2012_2/pdf/martin_hirst_RA_V6-2_2012_GMJAU.pdf)

Reflect on the following questions:

- What do you think about the author's view on social media?
- How does the limits of social media mentioned in the paper relate to your online survey?

## Submission
Respond to this post as a comment with at least 2 to 3 sentences answering the questions above.

**Comment by okcirE** [comment]
In a past anthropology class, I vaguely remembered us discussing humans are distinctively more superior than any other species on Earth because we have language and the means to share information from culture to culture. The creation of social media platforms only propelled the possibility of the sharing of information across the country a reality. For example, we have automatic translators built into messaging applications that are accessible in different countries which tremendously decreases the language barrier that we once had. Therefore, these advancements allow for the rise of online activism through the usage of social media platforms. I agree with the author's statement that social media/technology can shape the actions, behavior, and decisions humans make. This statement can be evident with algorithms constantly putting out trending words that could be false on users' front page. 

I think it is dangerous for us to assume that most people in this world have access to technology, thus our online survey will most likely not be the best representation of our target population. Not everyone has access to social media and for those who do, not all of them are honest people, which sucks. Our survey responses should be filtered, but we need to be careful of our own biases and develop greater discernment.

**Reply by albertkun** [reply]
@okcirE AHHHHH! Thanks for completing the extra credit reading!!! I really didn't hope you stayed up late for it!!! But I hope you found the article interesting! Good point about translations, which I agree does lower the language barrier. Of course, remember to ask who is determining the translations that are "accurate" versus actual people. I mention this because language is constantly changing so it goes back into inherent bias, but this time for meanings! For example, a machine translation is unable to predict meanings in the future for words, let alone the local context of people that are not fed into that machine.

**Comment by jsxu315** [comment]
I agree with the author’s point that social media amplifies voices rather than creating something out of nothing, and that social media activism doesn’t necessarily translate to substantive change. Anecdotally, I feel like COVID could have changed the situation a little; since we have to do a lot of things online now, it would seem that people have a reason to be more genuine online, including in online activism. 

While reading, I had trouble believing that technology never ‘takes on a life of its own’ so to speak; I thought that particularly with large movements, ripple effects in social media can go beyond what users intended. For example, when activists increase their reach over social media, they risk suppression by governments who are now more aware of them. (Due to the complexity of social/technological factors, though, I don’t think this leads to any kind of determinism.) After more thought, I think it might not actually conflict with the author’s ideas: the effects in the example are due to users’ (activists and government) conflicting interests, not of technology taking on a life of its own. 

I also really liked Hirst’s point about media determinism — publications have to utilize different technologies — social media, video, etc. in order to survive, which feeds into soft techno-determinism. And it’s easy to ascribe some power to technology (i.e. social media) when you see how your circulation/engagement tangibly changes with the tools that you use. I think techno-determinism in the media is here to stay, because the distancing of people from technology is so deeply ingrained in the modern age that people might not even realize it. Interviewing and questioning is also the main way that journalists gather info, so after finishing up all their interviews, a journalist might not think that there’s still some critical information missing; in fact, I wouldn’t have thought about techno-determinism if not for this article.

Overall I agree that what social media can do depends on the people that make/use it, but at a certain point, it might not matter that the technology itself doesn’t hold power if people believe that it does. By placing importance on the product instead of the people, people give up their power and are less able to make change (which just benefits the powerful and wealthy; not by accident?). 



Even if our survey is shared a lot, we might not get a lot of substantive responses, whether due to apathy or people lacking the technological resources. We also shouldn’t see our map as something that can accomplish our objects in and of itself — it’s a medium through which people can tell their stories, so the design and distribution of the survey should be placed on equal or higher importance to the coding of the map.

**Reply by albertkun** [reply]
@jsxu315 OMG JESS! Same as my response to Ericko... I hope you aren't staying up doing the extra credit reading!!! It seems like based on your response though you enjoyed the article! I am glad you did as it really did a better job of breaking down the techno-deterministic approach, especially as it relates to social media. It was quite a long article, but seeing your response really made me glad that I was able to still expose people to it since the type of thorough analysis Hirst makes on media and technology are often overlooked. I like how you explained the nuances of the author's approach and how you reflected on it as you went further. A very enjoyable response!! 

Also, I'm very glad with how you were able to connect the reading to your project well. It was great how you summarized the goal of your project well as a medium! I totally like that approach!

**Comment by annyrviloria** [comment]
It was easy for me to agree with the author's view on social media. In my view, we love social media because our brains are obsessed with the dopamine response we get from fast and endless threads of information and so we carry it around wanting for it to be involved in every aspect of life. The author makes a compelling case for the socioecological and historical contexts that explain our incessant need to ascribe supernatural powers to social media -- namely, the fetishization of tools under capitalism. One additional layer of complexity offered by the author is that capitalist values and modes of production also inform the biases present in journalism and media reporting. My favorite quote that summarizes this point is found in page 2: 

> As long as Mubarak's regime was stable and compliant with US wishes in the region (support for Israel) internal opposition was not particularly newsworthy.

This resonates with my worldview because having been raised by union leaders I know very well that disrupting the profits of the capitalist is the most effective way to call attention to the workers' plight. I also know that the favorite tactic of anti-union efforts is creating distractions that feed into what the author calls 'perpetual present' and the 'bias of availability'.  All this to say, I see a lot of overlap between this super important article and what I know to be the reality in the world of workers' struggles pre/during/post-social media era.

Thinking about the online survey we are developing in class, the limits of social media should be understood when we envision the impact of our application. Although I would love to claim that the maps being created from my survey will call attention AND address the struggle for refugee and migrants' rights in the southern border of the US, that statement feeds into the technological determinism that the author describes in relation to the western understandings of the Arab Spring. In reality, my online survey is simply a tool that might inform the organizing and advocacy efforts of the generation upon generations of justice fighters. For once, it is imperative to understand the contexts in which my survey operates by being aware of the people's history of the border. But perhaps more importantly, it is critical to ensure that the shininess and simplicity of my survey does not reduce the deep roots and reach of xenophobia in the US border to a few events in time that simply need visibility to be resolved.

**Comment by Wny-Duong** [comment]
It seems to me that Hirst's view on social media in the context of the social change in revolution is that it's another manifestation in an older trend of subversion of mainstream media and decentralized journalism, as opposed to being a driving force behind the promotion of actual change. There's a lot of evidence and papers he cites in reference to views on technological determinism and he himself has a pretty critical view of the notion when it comes to driving journalism, and suggests the idea of "soft determinism" to be the prevailing opinion in the academic field itself, despite it having a tendency to oversimplify the nuances of what "technology" means in the context of leading social change.

In the context of journalism itself, I think his views makes a whole lot of sense. The stories that most people seek to consume through social media suffer from the lack of attention that most people are willing to provide to those stories as a result of the whole information overload associated with modern social media. It makes sense that most journalists, therefore, would want to create and publish narratives that are more digestible. For instance, the idea of singular Arab Spring being attributed to social media creates a much clearer and more interesting narrative to read in a short burst than discussing the actual historic steps and instances of social unrest that boiled into many separated but similar instances of upheaval in Northern Africa and the Middle East. In this case, technological determinism is a narrative of its own making that primarily exists not because it is a driving point of social change, but because it's a simple and understandable way of explaining it, even when it's not actually the case. 

A few other people commenting in this discussion post have mentioned that social media gives a platform to viewpoints and provides an opportunity for people to voice opinions for change, but that those opinions and voices don't just manifest because they are given a platform. A proper investigation of events and stories sourced from the people originating those opinions would paint a far more accurate and meaningful picture of the backdrop of social revolution than investigating the digital megaphones they're using to incite change.

As for how the limits of social media reflect our online survey, there's honestly a lot that surveys can't really cover. The article already mentions the lack of technologies available to certain people despite the lower barriers of access than in previous years. [Even in California, we still see problems regarding the inequity of internet access among the population of the state. ](https://edsource.org/2020/interactive-map-internet-subscription-rates-in-california/629669). Without a proper understanding of the people you're surveying and real-world communication that promotes productive discussion, pretty much anyone creating a survey is bound to face the problem of the bias of convenience since they're only able to get information that they know to ask for. Reading this article actually made me realize the importance of why Albert wanted us to have the option to do proper groundwork in collecting data if this class was in-person in the first place. It's very hard to know what information you're actually lacking if you're only driving a discussion like a one-sided interrogation of "What are your opinions on this?", "How do you do this?", and "Why do you do that?" that assumes you're trying to acquire information to solve a problem that you decided had an answer that you only need additional information to back up or modify until it solves the problem that you decided was a problem worth solving.

EDIT: Wanted to make a point on focusing on technology oversimplifying, if you zoom in on the map I linked to UCLA, you'll notice that 0% of residents of the area have access to broadband. If you think about this and why it's weird, you'll realize that ~~nobody actually makes the university their primary residence because it's kind of not a home residence~~ (edit: the data is actually weird, there's 8 households and a population of about 10k. I would assume that this is because of the faculty in residence program and the student dorming population that don't have their own broadband subscription due to relying on UCLA's wifi but I've already been wrong before on this topic) , but you wouldn't know that unless you actually knew what that specific location on the map was and it'd look weird otherwise.
